recipes/logfault/all/conanfile.py

```diff
@@ -0,0 +1,135 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "logfault"
+    description = "Simple to use, header only C++ library for application-logging on all major platforms."
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case it's not listed there, use "LicenseRef-<license-file-name>"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jgaa/logfault"
+    # Do not put "conan" nor the project name in topics. Use topics from the upstream listed on GH
+    # Keep 'header-only' as topic
+    topics = ("logging", "header-only")
+    package_type = "header-library"
+    # Keep these or explain why it's not required for this particular case
+    settings = "os", "arch", "compiler", "build_type"
+    # Do not copy sources to build folder for header only projects, unless you need to apply patches
+    no_copy_source = True
+    generators = "CMakeToolchain", "CMakeDeps"
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    # In case the project requires C++14/17/20/... the minimum compiler version should be listed
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "15",
+            "clang": "15",
+            "gcc": "13",
+            "msvc": "14.31",
+            "Visual Studio": "17.1",
+        }
+    
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables['LOGFAULT_BUILD_TESTS'] = not self.conf.get("tools.build:skip_test", default=True, check_type=bool)
+
+    # Use the export_sources(self) method instead of the exports_sources attribute.
+    # This allows finer grain exportation of patches per version
+    def export_sources(self):
+        export_conandata_patches(self)
```

Comment:
```suggestion
    # Use the export_sources(self) method instead of the exports_sources attribute.
    # This allows finer grain exportation of patches per version
    def export_sources(self):
        export_conandata_patches(self)
```

Not needed if there are no patches